src/pkg/packaging/windows/GetWix.ps1

```diff
@@ -1,5 +1,5 @@
 ﻿Param(
-    [string]$WixFilename="wix.3.10.2.zip",
+    [string]$WixFilename="wix.3.10.4.zip",
```

Comment:
Can we just remove the default value, so this version number isn't copied in multiple places? This parameter is always specified when we call it from MSBuild. It would be nice to have a single place (the `package.props` file) that defines the version.  That way they can never be out of sync.